astropy/table/info.py

```diff
@@ -0,0 +1,97 @@
+"""
+Table method and functions related to providing information about
+columns and tables.
```

Comment:
I think this module is now only for tables, correct?  If so, correct docstring here as well as the docstring of `info` below.